CONTRIBUTING.md

```diff
@@ -1,31 +1,129 @@
+- [Dart Package](https://github.com/VeryGoodOpenSource/very_good_dart_package)
+- [Docs Site](https://github.com/VeryGoodOpenSource/very_good_docs_site)
+- [Flame Game](https://github.com/VeryGoodOpenSource/very_good_flame_game)
+- [Flutter Starter App (Very Good Core)](https://github.com/VeryGoodOpenSource/very_good_core)
+- [Flutter Package](https://github.com/VeryGoodOpenSource/very_good_flutter_package)
+- [Flutter Plugin](https://github.com/VeryGoodOpenSource/very_good_flutter_plugin)
+- [Flutter Wear OS App](https://github.com/VeryGoodOpenSource/very_good_wear_app)
 
-Before creating a pull request please:
+If there are additional templates you'd like to see, open an issue and tell us!
 
-1. Fork the repository and create your branch from `main`.
-1. Install all dependencies (`flutter packages get` or `pub get`).
-1. Squash your commits and ensure you have a meaningful, [semantic][conventional_commits_link] commit message.
-1. Add tests! Pull Requests without 100% test coverage will not be approved.
-1. Ensure the existing test suite passes locally.
-1. Format your code (`dart format .`).
-1. Analyze your code (`dart analyze --fatal-infos --fatal-warnings .`).
-1. Create the Pull Request.
-1. Verify that all status checks are passing.
+💡 Note: Very Good CLI's completion functionality is powered by [CLI Completion](https://github.com/VeryGoodOpenSource/cli_completion) and its test command optimization is powered by [test optimizer](bricks/test_optimizer/README.md). If you want to contribute to either of those, please refer to their respective repositories.
 
-While the prerequisites above must be satisfied prior to having your
-pull request reviewed, the reviewer(s) may ask you to complete additional
-work, tests, or other changes before your pull request can be ultimately
-accepted.
+## Opening an issue
+
+We highly recommend [creating an issue][bug_report_link] if you have found a bug, want to suggest a feature, or recommend a change. Please do not immediately open a pull request. Opening an issue first allows us to reach an agreement on a fix before you put significant effort into a pull request.
+
+When reporting a bug, please use the built-in [Bug Report][bug_report_link] template and provide as much information as possible including detailed reproduction steps. Once one of the package maintainers has reviewed the issue and we reach an agreement on the fix, open a pull request.
+
+## Developing for Very Good CLI
+
+To develop for Very Good CLI you will need to become familiar with Very Good Ventures processes and conventions:
+
+### Setting up your local development environment
+
+1. Install a valid [Dart SDK](https://dart.dev/get-dart) in your local environment. Compatible Dart SDK versions with test optimizer can be found [here](https://github.com/VeryGoodOpenSource/very_good_cli/blob/66c2a5e847ba4db6a9cb212e2b08e2af9f9e70d2/pubspec.yaml#L7). If you have Flutter installed, you likely have a valid Dart SDK version already installed.
```

Comment:
👀  should this not refer to main?

Reply:
I've changed this, let me know if you're happy with the new change by resolving this comment.